src/dht_utils.py

```diff
@@ -72,34 +72,63 @@ async def _declare_active_modules(
     )
 
 
+def get_remote_sequence(
+    dht: DHT,
+    start: int,
+    stop: int,
+    config: src.DistributedBloomConfig,
+    dht_prefix: Optional[str] = None,
+    return_future: bool = False,
+) -> Union[src.RemoteSequential, MPFuture]:
+    return RemoteExpertWorker.run_coroutine(
+        _get_sequence_blocks(dht, start, stop, config, dht_prefix), return_future=return_future
+    )
+
+
+async def _get_sequence_blocks(
```

Comment:
```suggestion
async def _get_remote_sequence(
```

In the rest of file, private coroutines have the same name as public non-async functions. Let's stick to that in the new functions as well.

Reply:
applied as a separate commit